ca/ca.go

```diff
@@ -32,8 +32,9 @@ type CAImpl struct {
 }
 
 type issuer struct {
-	key  crypto.Signer
-	cert *core.Certificate
+	key    crypto.Signer
+	keyRaw *rsa.PrivateKey
```

Comment:
I think you can get away without having this `keyRaw` field by making a [type switch](https://github.com/golang/go/wiki/Switch#type-switch) on the `key` field in the location you'd be calling `GetIntermediateKey`, looking for a `rsa.PrivateKey` type (presently the only kind of key that Pebble will generate).

Reply:
In fact I tried the type checking, but the compiler refuse it because PrivateKey is not implementing Signer. The doc says that this interface is designed to be opaque, so I do not know how to go through that.

Reply:
Ok nevermind. I was trying to cast to the interface crypto.PrivateKey instead of the actual implementation rsa.PrivateKey. The type switch is working now.